rust-netlink/netlink-packet-route
# support IFLA_VRF_PORT_TABLE attribute

Links that belong to a VRF carry the VRF table ID in the link info. For a VRF link we already had these attributes:

```
LinkInfo([Kind(Vrf), Data(Vrf([TableId(10)]))])
```

For ports attached to a VRF we now have:

```
LinkInfo([Kind(Veth), PortKind(Vrf), PortData(VrfPort([TableId(10)]))])
```

This tells us that this veth interface belong to the VRF with table ID 10.

----

To test this:

```
sudo ip link add vrf-blue type vrf table 10                                                            
sudo ip link set dev vrf-blue up                                                                         
sudo ip link add veth20 type veth
sudo ip link set veth20 master vrf-blue # this link should now have the new attribute
```

Ref: https://github.com/torvalds/linux/blob/f2e367d6ad3bdc527c2b14e759c2f010d6b2b7a1/drivers/net/vrf.c#L1797
